### PR TITLE
Improve styling of website

### PR DESCRIPTION
**Pull request by CDE90**
Me again... sorry!

This PR just changes the styling of the website and diff viewer

Currently it's not in the t3 colours - just boring grey - but this can obviously be changed if needed

Here's a preview:
image

### CODE REVIEW

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Html, Main, NextScript, Head } from "next/document";
+
+const MyDocument = () => {
+  return (
+    <Html>
+      <Head>
+        <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
```
Comment (andreifilip123) [inline src/pages/_document.tsx]: Please change this to use next-font instead:

https://nextjs.org/docs/basic-features/font-optimization#google-fonts

File: src/pages/diff/[slug].tsx [hunk 2]
```diff
@@ -166,30 +173,151 @@ export const getStaticProps: GetStaticProps<Props, Params> = async (
...
+            <li
+              key={feature}
+              className={`${
+                enabled ? "bg-green-500" : "bg-gray-500"
+              } m-2 rounded-lg p-3 font-bold text-white shadow-lg`}
+            >
+              {feature}
+            </li>
+          )
+        )}
```
Comment (andreifilip123) [inline src/pages/diff/[slug].tsx]: Not sure how but I'd really like to improve these 🙈 I really dislike the look of these

**Review by andreifilip123 — CHANGES_REQUESTED** [review]
Looks really good overall. Just some minor changes.

Thank you very much for your help on this !

File: src/pages/_document.tsx (hunk 1 above)
Comment (CDE90) [reply src/pages/_document.tsx]: Done in the latest commit

File: src/pages/diff/[slug].tsx (hunk 2 above)
Comment (CDE90) [reply src/pages/diff/[slug].tsx]: How about something like this?
image

Comment (andreifilip123) [reply src/pages/diff/[slug].tsx]: Looks good

**Review by andreifilip123 — APPROVED** [review]

**Pull request merged**